.travis.yml

```diff
@@ -100,16 +100,17 @@ script:
   - git submodule update --quiet --init --recursive
   - echo 'Building POSIX Firmware..' && make posix_sitl_default
   - echo 'Running Tests..' && make posix_sitl_default test
+  - build_configs="px4fmu-v1_default px4fmu-v2_default px4fmu-v2_lpe \
+        mindpx4-v2_default uavcan_firmware \
+        px4fmu-v4_default px4-stm32f4discovery_default"
   - if [ "${TRAVIS_OS_NAME}" = "linux" ]; then
          cd ${TRAVIS_BUILD_DIR}
       && make check_format
       && arm-none-eabi-gcc --version
-      && echo 'Building NuttX px4fmu-v1 Firmware..' && make px4fmu-v1_default
-      && echo 'Building NuttX px4fmu-v2 Firmware..' && make px4fmu-v2_default
-      && echo 'Building NuttX mindpx-v2 Firmware..' && make mindpx-v2_default
-      && echo 'Building UAVCAN node firmware..' && make uavcan_firmware
-      && echo 'Building NuttX px4fmu-v4 Firmware..' && make px4fmu-v4_default
-      && echo 'Building NuttX px4-stm32f4discovery Firmware..' && make px4-stm32f4discovery_default
+      && build_configs="px4fmu-v1_default px4fmu-v2_default px4fmu-v2_lpe \
```

Comment:
Yes we don't need to make your PR more complicated. I'll change it later.
Personally I just want to be able to catch problems locally before they
show up in travis.

On Thu, Apr 14, 2016 at 12:08 PM, James Goppert <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/PX4/Firmware/pull/4196#discussion_r59744772>:
>
> >    - if [ "${TRAVIS_OS_NAME}" = "linux" ]; then
> >           cd ${TRAVIS_BUILD_DIR}
> >        && make check_format
> >        && arm-none-eabi-gcc --version
> > -      && echo 'Building NuttX px4fmu-v1 Firmware..' && make px4fmu-v1_default
> > -      && echo 'Building NuttX px4fmu-v2 Firmware..' && make px4fmu-v2_default
> > -      && echo 'Building NuttX mindpx-v2 Firmware..' && make mindpx-v2_default
> > -      && echo 'Building UAVCAN node firmware..' && make uavcan_firmware
> > -      && echo 'Building NuttX px4fmu-v4 Firmware..' && make px4fmu-v4_default
> > -      && echo 'Building NuttX px4-stm32f4discovery Firmware..' && make px4-stm32f4discovery_default
> > +      && build_configs="px4fmu-v1_default px4fmu-v2_default px4fmu-v2_lpe \
>
> @dagar <https://github.com/dagar> I changed the order so that px4fmu-v4
> and uavcan_firmware are last. I do not believe the for loop is functionally
> different from the current setup since it will still abort on the first
> error and will maintain the order. I'm ok with moving the target to the
> Makefile if you think that would be more apropriate, but I think we can
> wait on that and not include the change in this PR.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/PX4/Firmware/pull/4196/files/f9ddbd828f1b0e5a842af6e705d67a4c09542eeb#r59744772>
>